### PR TITLE
Update win_dns_client.py

### PR DESCRIPTION
**Pull request by alexzheng0000**
If there is no dns address setted in the windows client, the iface_config.DNSServerSearchOrder method will return None, therefore this module will generate TypeError,  'NoneType' object is not iterable.

### CODE REVIEW

File: salt/modules/win_dns_client.py [hunk 1]
```diff
@@ -45,7 +45,10 @@ def get_dns_servers(interface='Local Area Connection'):
         for iface in c.Win32_NetworkAdapter(NetEnabled=True):
             if interface == iface.NetConnectionID:
                 iface_config = c.Win32_NetworkAdapterConfiguration(Index=iface.Index).pop()
-                return list(iface_config.DNSServerSearchOrder)
++                try:
```
Comment (jfindlay) [inline salt/modules/win_dns_client.py]: @cheng0919, it looks like you forgot to remove the `+` diff chars at the beginning of the lines.  Also, the `except` will need to be `except TypeError`.  Thanks for fixing this.

**Comment by jfindlay** [comment]
@cheng0919, just one more [lint error](https://jenkins.saltstack.com/job/salt-pr-lint-n/8768/violations/file/salt/modules/win_dns_client.py/).

**Comment by jfindlay** [comment]
Fixes #26626.

**Comment by cheng0919** [comment]
@jfindlay  Sorry for this omission :)

**Comment by jfindlay** [comment]
@cheng0919, no problem and thanks for your fix.

**Comment by basepi** [comment]
This is great, thanks!

**Pull request merged**